apps/application-system/api/src/app/modules/application/files/utils/constants.ts

```diff
@@ -26,3 +26,10 @@ export const BucketTypePrefix: KeyMapping<PdfTypes, string> = {
 export const DokobitFileName: KeyMapping<PdfTypes, string> = {
   ChildrenResidenceChange: 'Lögheimilisbreyting-barns.pdf',
 }
+
+export const DokobitErrorCodes: { [key: string]: number } = {
```

Comment:
Do we need to define the type here? 
```suggestion
export const DokobitErrorCodes = {
```